pkgs/by-name/gi/github-runner/package.nix

```diff
@@ -80,19 +80,16 @@ buildDotnetModule rec {
       url = "https://github.com/actions/runner/commit/5ff0ce1.patch";
       hash = "sha256-2Vg3cKZK3cE/OcPDZkdN2Ro2WgvduYTTwvNGxwCfXas=";
     })
+    # Fix source path discovery in tests
+    ./patches/test-getsrcpath.patch
   ];
 
-  postPatch =
-    ''
-      # Ignore changes to src/Runner.Sdk/BuildConstants.cs
-      substituteInPlace src/dir.proj \
-        --replace 'git update-index --assume-unchanged ./Runner.Sdk/BuildConstants.cs' \
-                  'true'
-    ''
-    + lib.optionalString (nodeRuntimes == [ "node20" ]) ''
-      substituteInPlace src/Runner.Common/Util/NodeUtil.cs \
-        --replace-fail '_defaultNodeVersion = "node16"' '_defaultNodeVersion = "node20"'
-    '';
+  postPatch = ''
+    # Ignore changes to src/Runner.Sdk/BuildConstants.cs
+    substituteInPlace src/dir.proj \
+      --replace 'git update-index --assume-unchanged ./Runner.Sdk/BuildConstants.cs' \
```

Comment:
```suggestion
      --replace-fail 'git update-index --assume-unchanged ./Runner.Sdk/BuildConstants.cs' \
```

Reply:
I didn't actually change these lines, and there are plenty places in this file with `--replace`, but ok, I fixed this one instance.